### ISSUE TITLE
Direct link to meeting

### ISSUE THREAD

**Issue by piccolbo** [jitsi/jitsi-meet#10469]
**Is your feature request related to a problem you are facing?**

We are trying to switch family to jitsi meet for weekly calls. We need low friction path for sweet old grandma with android tablet and no local help. When she taps on the meeting link, she gets a screen full of English text that she can't understand. If we manage to get her to tap on the right button (this on the phone, talking about a screen, as fun as pulling teeth), now she has to type her name (as if we didn't know) and press join meeting. This is an accessibility issue.

**Describe the solution you'd like**

Some additional URL param like `https://meet.jit.si/familylove?grandmamode=yes` that skips any intermediate screens and goes to the call no-questions-asked, no clicks, no  nothing. 1 click to call. Or make that `skipappinterstitial=true` or whatever is more consistent with your route design and parameter naming. 

**Describe alternatives you've considered**

1. Educating old person to jump through the hoops. We have to retrain every weekend over the phone. Doesn't stick. Typing on tiny keyboard is extra hard, training or not, for old hands
2. Begging neighbor to go and initiate meeting for old grandma. Can't impose on neighbor every weekend

**Commentary**

I would like to create a sort of jitsi terminal or kiosk. You turn the thing on and you are in a meeting. The shorter the path to achieve this goal the better.  Think of all the low tech grandmas out there that you'll make happy. Thanks!

**Comment by damencho** [comment]
Please, when you have questions or problems use the community forum before opening new issues, thank you.

This is already possible use the following link:
`https://meet.jit.si/familylove#config.prejoinPageEnabled=false`

**Comment by piccolbo** [comment]
It does work on desktop, but on Android one is still stuck on some other page that gives an option of joining in app, downloading the app or launching in browser, with lots of other information, phone numbers etc.  and Android is the relevant platform for my feature request. I didn't think of using a forum, but I looked for support, contact or documentation links before tracking the project down on GH. I couldn't see any. On the other hand, I saw links to terms and conditions and privacy policy in small font at the bottom of the jitsi home page, so it's not like I didn't look. Also, in the meeting screen I didn't see an help or support option. Maybe it should be made available or  more prominent so that people can try those options before coming here on github. I am a developer  so I understand your point about using the appropriate channels, but you have to make those channels visible.

**Comment by saghul** [comment]
On mobile browsers we give the option of joining via the browser or launching the app directly.

Are you having any problems with this?

**Comment by piccolbo** [comment]
Correct, that is the problem. I understand that most users can navigate that page without problem. You are an international company, so I am sure it's not news to you that there are people that don't speak a word of English. Now if you add to that that we are talking about an older user with minimal familiarity with computers, that page becomes an obstacle, sometimes insurmountable because there is no local help available and all support has to happen over the phone.  The page that you describe as offering a binary choice has three buttons and about 25 lines of information, including the list of call in numbers. Now you may say that you are trying to serve as many users as you can but there is a limit where it's beyond what you can do. I know, I've had helpless users myself. How far can one dumb it down? But the direct link to   video feature exists on desktop, as @damencho showed. If only Android behaved consistently, that would be a solved problem. Why not?

**Comment by saghul** [comment]
It's not possible to solve it the way you expect, at least not to my knowledge.

On Android you need a special type of link which would open the app, that is what the "launch in app" button does. The problem is, we can't know if you have the app installed or not so if the page would try to open the app and failed, you'd see nothing, because it's not a regular HTTP link.

So that's why you get the choice.

Something you can do if you need a shortcut to a fixed meeting name is to create a shortcut to the intent URL (copy the target of the launch in app button) and use that. There are a bunch of URL shortcut creators in the Play Store.
